test/debugger/stepout-test1.js

```diff
@@ -0,0 +1,26 @@
+/*
+Basic test for stepOut
```

Comment:
How does the test work? Do we have any test runner to verify this step out behavior from code? Or it is just a JS code for manual testing? If so, I do not think it makes sense to checkin.

Reply:
Currently it's just manual -- I was going to accumulate them until I had a chance to write a test runner. Will remove from commits until the runner is made